custom_components/tuya_local/devices/fisher_summer_air_conditioner.yaml

```diff
@@ -0,0 +1,463 @@
+        #   2. Whether the temperature is adjustable in automatic mode
+        #   3. Fresh air volume mark
+        #   4. Vector air supply
+        #   5. Left and right sweeping air
+        #   6. Photosensitive
+        #   7. Intelligent dehumidification and anti-mildew
+        #   8. Humidity sensor
+        #   9. Evaporator cleaning
+        #   10. Save money and see it
+        #   11. Power statistics
+        #   12. Generator mode
+        #   13. High temperature wind/cool wind 
+        #   14. Air quality detection function
+        #   15. Set to empty (original: humidity function)
+        #   16. Set to empty (original: equipment operation saves money and visible temperature curve display)
+        #   17. 8℃ heating
+        #   18. Filter dirty and clogged function
+        #   ??? - 19 is missing in Tuya json
+        #   20. presence or absence of PM2.5
+        #   21. temperature scale switching, 1 is Fahrenheit, 0 is Celsius
+        #   22. soft wind
+        #   23. left and right wide-angle air supply
+        readonly: true
+  - entity: select
+    name: Vertical Sweep
+    category: config
+    dps:
+      - id: 113
+        name: option
+        type: string
+        mapping:
+          - dps_val: 0
+            value: None
+            default: true
+          - dps_val: 1
+            value: Upper and lower
+          - dps_val: 2
+            value: Upper
+          - dps_val: 3
+            value: Lower
+  - entity: select
+    name: Horizontal Sweep
+    category: config
+    dps:
+      - id: 114
+        name: option
+        type: string
+        mapping:
+          - dps_val: 0
+            value: None
+            default: true
+          - dps_val: 1
+            value: Left and Right
+          - dps_val: 2
+            value: Left
+          - dps_val: 3
+            value: Middle
+          - dps_val: 4
+            value: Right
+          - dps_val: 5
+            value: Partial Left
+          - dps_val: 6
+            value: Partial Right
+          - dps_val: 7
+            value: Wide Angle
+  - entity: select
+    name: Energy Saving
+    category: config
+    dps:
+      - id: 119
+        name: option
+        type: string
+        mapping:
+          - dps_val: 0
+            value: None
+            default: true
+          - dps_val: 1
+            value: Default Power Saving
+          - dps_val: 2
+            value: Quota Power Saving
+          - dps_val: 3
+            value: Fixed Temperature Power Saving
+        optional: true
+  - entity: select
+    name: Generator Mode
+    category: config
+    dps:
+      - id: 120
+        name: option
+        type: string
+        mapping:
+          - dps_val: off
+            value: None
+            default: true
+          - dps_val: L1
+            value: L1
+          - dps_val: L2
+            value: L2
+          - dps_val: L3
+            value: L3
+        optional: true
+  - entity: sensor 
+    name: Boolcode Options
```

Comment:
This one is probably also not useful as a separate sensor and can just be an extra attribute.